### ISSUE TITLE
what are the eq_coeffs?

### ISSUE THREAD

**Issue by bhazelton** [HERA-Team/hera_corr_cm#16]
I think I have an idea, that their job is to keep the bit statistics in a happy range, but looking at the actual values, they appear to be the same across the band for any given antenna, which I would not expect, and to be wildly different per antenna, which I found mildly surprising.

Also, where are these divided back out?

**Comment by realtimeradio** [comment]
Traditionally in PAPER no-one has ever had much enthusiasm for doing per-frequency channel EQ, which is why the coeffs don't vary over the band. They could, but they don't.
Input ADC power levels (analog variable attenuation) aren't tuned too precisely, and are sensitive to RFI, so the overall power going in to the system between different antennas isn't very consistent. Also, looking at the auto-corrs coming out of the correlator, the power levels have a fair bit of variation between ants, so the chosen EQ coeffs clearly aren't the best ones.
Basically, more care needs to be applied.

Coefficients aren't divided out anywhere in the DSP chain.

**Comment by jack-h** [comment]
Traditionally in PAPER no-one has ever had much enthusiasm for doing per-frequency channel EQ, which is why the coeffs don't vary over the band. They could, but they don't.
Input ADC power levels (analog variable attenuation) aren't tuned too precisely, and are sensitive to RFI, so the overall power going in to the system between different antennas isn't very consistent. Also, looking at the auto-corrs coming out of the correlator, the power levels have a fair bit of variation between ants, so the chosen EQ coeffs clearly aren't the best ones.
Basically, more care needs to be applied.

Coefficients aren't divided out anywhere in the DSP chain.

**Comment by bhazelton** [comment]
Fascinating. Thank you for the context.
